app/controllers/home_controller.rb

```
@@ -20,28 +29,94 @@ def logout
+  	@user = User.authenticate(user_params[:username], user_params[:password])
+    
   	if(@user) # if user found
   		session[:user_id] = @user.id
-  		redirect_to home_index_path
-  	else # if user not found
+      respond_to do |format|
+        format.html { redirect_to home_index_path }
+    		format.json { render json: @user }
+        flash.now[:notice] = @user.username || "Test"
+
+      end
```

RuboCop (1.76.1)

[convention] Extra empty line detected at block body end.

(Layout/EmptyLinesAroundBlockBody)